content/webapp/components/WorksRelatedContentCard/WorksRelatedContentCard.styles.tsx

```diff
@@ -0,0 +1,103 @@
+import styled, { css } from 'styled-components';
+
+import { font } from '@weco/common/utils/classnames';
+
+const clampLineStyles = css<{ $linesToClamp: number }>`
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  -webkit-line-clamp: ${props => props.$linesToClamp};
+`;
+
+const minGridCellWidth = '400px';
+const maxTextWrapperWidth = '270px';
+
+export const Card = styled.a`
+  display: flex;
+  padding: ${props => props.theme.spacingUnits['3']}px;
+  background-color: ${props => props.theme.color('white')};
+  border-radius: ${props => props.theme.borderRadiusUnit}px;
+  flex-wrap: wrap;
+  text-decoration: none;
+
+  @container (min-width: ${minGridCellWidth}) {
+    height: 10rem;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+  }
+`;
+
+export const TextWrapper = styled.div`
+  display: flex;
+  padding-top: ${props => props.theme.spacingUnits['5']}px;
+  flex-direction: column;
+  width: 100%;
+  container-type: inline-size;
```

Comment:
this lets us decide whether we need to limit the width of the labels (mostly because 'Archives and manuscripts' is much longer than the others) and lets us decide based on the available space (might be larger if e.g. there's a portrait image or no image at all) rather than based on the viewport